portal/app/[locale]/genesis-drop/_components/claimRewardsDisabledTestnet.tsx

```diff
@@ -13,7 +13,7 @@ export const ClaimRewardsDisabledTestnet = function () {
   return (
     <div className="absolute inset-0 flex flex-col items-center justify-center gap-y-1">
       <LiveIcon />
-      <h2 className="text-lg font-semibold text-neutral-950">
+      <h2 className="text-2xl font-semibold text-neutral-950">
```

Comment:
Per [globals.css](https://github.com/hemilabs/ui-monorepo/pull/1577/files#diff-c6d9ef62ac5f8bfe3a3b4771ac7d2e276ae0a5c19747747061e5ef779f9dbf18) these styles are redundant

```suggestion
      <h2 className="text-neutral-950">
```